octavia/api/app.py

```diff
@@ -106,6 +111,21 @@ def _wrap_app(app):
                 reason=e
             )
 
+    # OSProfiler tracing middleware
+    if CONF.profiler.enabled and profiler_initializer and profiler_opts and profiler_web:
```

Comment:
`CONF.profiler.enabled` config option is lazy-registered by import `osprofiler.opts`, so this check order will cause an exception if the library is missing:

```python
Traceback (most recent call last):
  File "/Users/d072895/Library/Application Support/JetBrains/Toolbox/apps/PyCharm-P/ch-0/203.6682.179/PyCharm.app/Contents/plugins/python/helpers/pydev/pydevd.py", line 1485, in _exec
    func()
  File "octavia/cmd/api.py", line 33, in main
    app = api_app.setup_app(argv=sys.argv)
  File "octavia/api/app.py", line 77, in setup_app
    wsme=pecan_config.wsme
  File "/Users/d072895/.virtualenvs/stein_python2.7/lib/python2.7/site-packages/pecan/__init__.py", line 91, in make_app
    app = wrap_app(app)
  File "octavia/api/app.py", line 115, in _wrap_app
    if CONF.profiler.enabled and profiler_initializer and profiler_opts and profiler_web:
  File "/Users/d072895/.virtualenvs/stein_python2.7/lib/python2.7/site-packages/oslo_config/cfg.py", line 2187, in __getattr__
    raise NoSuchOptError(name)
oslo_config.cfg.NoSuchOptError: no such option profiler in group [DEFAULT]
python-BaseException
Exception KeyboardInterrupt in <module 'threading' from '/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/threading.pyc'> ignored
```